### PR TITLE
Added announcements and jobposting workflows

### PR DESCRIPTION
**Pull request by JohnRoux**

### CODE REVIEW

**Comment by ChristianWitts** [comment]
As the workflow file is just json, could you run those through a prettyfier?

File: jobpostings_bot.slackworkflow [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        },
+        {
+          "type": "message",
+          "id": "407c1bfa-dc44-43b1-9221-e2c35c0bd6e9",
+          "config": {
+            "channel": {
+              "value": "C1RAZB24S"
+            },
+            "has_button": false,
+            "message_text": ":open_file_folder: New Jobposting Submission from {{424270b5-094d-4566-a304-b352c7d907e7==user}}\n\n:city_sunset: *Company Name*: {{6dd7eb07-cb63-4ee7-ab0b-ff66b6558605==6960dffd-779d-4b84-a6ee-bea523e3ee80==text}}\n\n:world_map: *Location:* {{6dd7eb07-cb63-4ee7-ab0b-ff66b6558605==a3bc14c5-e47e-4edc-8825-59ce44fe570c==text}}\n\n:walking: *Remote Friendly:* {{6dd7eb07-cb63-4ee7-ab0b-ff66b6558605==1e7231f6-352f-48c9-806b-e53bee484691==text}}\n\n:moneybag: *Salary Info:* {{6dd7eb07-cb63-4ee7-ab0b-ff66b6558605==d0e35551-b4b2-4363-ba57-93e1995d9c79==text}}\n\n:nerd_face: *Contact Person:* {{6dd7eb07-cb63-4ee7-ab0b-ff66b6558605==4d82930f-ae39-4ed5-86b0-d562ecd6a688==text}}\n\n:desktop_computer: *Job Description:*\n{{6dd7eb07-cb63-4ee7-ab0b-ff66b6558605==5b944ac2-66b3-4e41-b036-0a9f9160f5a6==text}}\n\n\n:link: *Additional Link:* {{6dd7eb07-cb63-4ee7-ab0b-ff66b6558605==90ba754d-61e7-4455-bde4-4c78ee946102==text}}"
```
Comment (JohnRoux) [inline jobpostings_bot.slackworkflow:155]: Oof

**Review by ChristianWitts — APPROVED** [review]
Woohoo, awesome :shipit:

File: jobpostings_bot.slackworkflow (hunk 1 above)
Comment (ChristianWitts) [reply jobpostings_bot.slackworkflow:155]: Yeah, not the prettiest, maybe json6 will bring multiline strings in... :joy:

**Pull request merged**